x/auth/client/cli/encode.go

```diff
@@ -10,6 +10,9 @@ import (
 	authclient "github.com/cosmos/cosmos-sdk/x/auth/client"
 )
 
+// TODO HV2: GetEncodeCommand is slightly different in heimdall.
+//  We need import the `helper` and apply the changes from heimdall's `client/tx/broadcast.go`
+
```

Comment:
I don't think we need to import `helper`. Instead we can directly port in the methods here (doesn't seem complex at a glance).

Reply:
True that, but I decided not to import it just yet as the helper function is also used by other packages (e.g. the `client` dir). So to avoid duplications, left a `TODO` to port/implement such functions when migration is completed.